bin/hummingbot.py

```diff
@@ -55,8 +55,6 @@ def detect_available_port(starting_port: int) -> int:
     await create_yml_files()
     read_configs_from_yml()
 
-    init_logging("hummingbot_logs.yml")
```

Comment:
Please don't delete this. It's there for ensuring proper logging during `HummingbotApplication()` initialization. The logging config is overwritten afterwards.

Reply:
Ok got it. I thought it was an error. I just put it back.